test/README.md

```diff
@@ -51,7 +51,6 @@ This should help locating the test in case the top of the stack can't be read du
 The error includes the logged message as well as the stacktrace of that message.
 Unfortunately the stacktrace is currently duplicated due to `chai`.
 
-However, in watchmode (`yarn test:unit --watch`) unexpected calls are ignored (see https://github.com/mochajs/mocha/issues/4347).
```

Comment:
@oliviertassinari `yarn test:watch` should fail on unexpected console warns/errors now.

Reply:
@dependabot squash and merge

Reply:
awesome :)